src/pages/gallery/elementIntro.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Introduction to PrairieLearn questions
```

Comment:
Let's consider calling this page `question-intro.md` instead of `elementIntro.md`, since it deals with much more than just elements. Ditto for the adjacent `elementIntro.tsx` file.